test/testenv/ingest_utils.go

```diff
@@ -184,7 +186,8 @@ func CopyFileToPod(podName string, srcPath string, destPath string, deployment *
 	if err != nil {
 		return "", "", err
 	}
-	restClient, err := apiutil.RESTClientForGVK(gvk, restConfig, serializer.NewCodecFactory(scheme.Scheme))
+	//FIXME
```

Comment:
nit: May be a better comment, if still pending work?

Reply:
removed FIXED, you are right I should have added comments with FIXME and TODO messages. will not down for future changes.